databases/week3/create_database.sql

```diff
@@ -0,0 +1,76 @@
+use meal_sharing;
+
+CREATE TABLE `reservation` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `number_of_guests` int(10) unsigned NOT NULL,
+  `meal_id` int(10) unsigned NOT NULL,
+  `created_date` DATE,
+  `contact_phonenumber` varchar(255) NOT NULL,
+  `contact_name` varchar(255) NOT NULL,
+  `contact_email` varchar(255) NOT NULL,
+  FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `meal` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `title` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
```

Comment:
The type here should should be `text`: https://dbdiagram.io/d/5f0460690425da461f045a29